docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -81,25 +106,44 @@ Depending on the logic of `myMethod()`, this could print something like the foll
-- `preconditions: { account: { state: [...] } }` – similar to the `update`, this has one entry per field of on-chain state. These are the preconditions that you created with `this.<state>.assertEquals()`. In this example, your transaction are accepted only if the first of the 8 state fields equals 0. The `null` values mean that there's no condition on the other 7 state fields.
-- `authorizationKind: 'Proof'` – this means that this account update needs to be authorized with a proof. This is the default when you call a zkApp method, but not necessarily for other account updates.
-- `authorization: undefined` – the proof that's needed on this update isn't there yet! You learn how to add it in a minute.
+Note that there are many more fields that account updates can have, but `tx.toPretty()` prints only the fields with actual content. Also, not all account updates must be present. For example, if a zkApp doesn't set a state, the `update` field might be missing. In that case, strictly speaking, it wouldn't always be an "update" in the sense that the account is modified. The term "account update" is used for simplicity.
 
-Note that there a many more fields that account updates can have, but `tx.toPretty()` only prints the ones that have actual content. Also, the ones above may be missing: For example, if our zkApp doesn't set any state, the `update` field might be missing. In that case, strictly speaking it wouldn't always be an "update" in the sense that the account is modified. The term "account update" is used for simplicity.
+As you might have noticed, account updates aren't created in a very explicit manner. Instead, o1js gives you an imperative API, with "commands" like `state.set()`, to create and modify account updates in a transaction. 
 
-As you might have noticed, these account updates weren't created in a very explicit manner. Instead, o1js gives you an imperative API, with "commands" like `state.set()`. Under the hood, these commands create and modify account updates in a transaction, like you saw above. In the end, the entire transaction is sent to the network, as one atomic update. If something fails – for example, one of the account updates has insufficient authorization – the _entire_ transaction is rejected and doesn't get applied. This is in contrast to an EVM contract, where the initial steps of a method call could succeed even if the method fails at a later step.
+In the end, the entire transaction is sent to the network as one atomic update. If something fails – for example, one of the account updates has insufficient authorization – the _entire_ transaction is rejected and doesn't get applied. This is in contrast to an EVM contract, where the initial steps of a method call could succeed even if the method fails at a later step.
 
-## Creating proofs and what they mean
+## Creating proofs
 
-Let's finally see how to create zero-knowledge proofs!
+To create zero-knowledge proofs:
```

Comment:
We use second person pronoun "you" so the r/w was to avoid using 'we' 
https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide#language-and-grammar 

I've put the fun back in!